lib/Fhp/Action/GetBalance.php

```diff
@@ -48,21 +48,41 @@ public static function create(SEPAAccount $account, bool $allAccounts = false):
         return $result;
     }
 
+    /**
+     * @deprecated Beginning from PHP7.4 __unserialize is used, then this method is never called
+     */
     public function serialize(): string
     {
-        return serialize([
-            parent::serialize(),
+        return serialize($this->__serialize());
+    }
+
+    public function __serialize(): array
+    {
+        return [
+            parent::__serialize(),
```

Comment:
I think this should be `serialize(parent)`, otherwise we're breaking backward compatibility with data serialized before this change. And analogous for the unserialization.

Reply:
Alternatively, you can keep your change here and make the unserialization detect whether it's receiving a string or an array -- that way we can eventually deprecate/remove the old format (nested arrays seem cleaner indeed).

Reply:
It might actually be worth creating an object instance on your machine somehow, commenting out the new magic functions for a second, getting the serialized string, and adding a unit test that only runs on PHP 7.4+ and verifies that this old serialized string can still be unserialized.

Reply:
I would not touch the __serialize methods, because we don't need backward compatibility there:
No one, will create a serialized string in an updated system to import them in an outdated system.

But i reviewed the __unserialized method in the way you suggested. 
I also added a test and found a bug, when the compatibility method "unserialize" is used in a nested scenario (parent/child).

So, if i not missed something - all should be done